projects/ponytaswap/index.js

```diff
@@ -1,11 +1,19 @@
 const { getUniTVL } = require('../helper/unknownTokens')
+const {staking} = require("../helper/staking");
 
+const pool = [
```

Comment:
hi @shengwenzheng 
what are these pools? when I go to https://www.ponytaswap.finance/ I dont see anything in the pools/farms page. Bug in the UI?

Reply:
Sorry, this is a mistake, I will delete the code later